### PR TITLE
Fastboot2

### LINKED ISSUE #68: Update to a more recent version of fastboot
Currently using 0.6

### PR DESCRIPTION
**Pull request by Gaurav0**
Fixes #68

### CODE REVIEW

File: bin/ember-fastboot [hunk 1]
```diff
@@ -24,53 +24,60 @@ if (!distPath) {
       "  ember-fastboot fastboot-dist");
   process.exit(1);
 }
+//
```
Comment (toddjordan) [inline bin/ember-fastboot]: can we just delete all the commented code?

Comment (Gaurav0) [inline bin/ember-fastboot]: Well I disabled the redis caching feature for now. I could delete the code associated with it but I'd prefer to figure out how to reenable it.

Comment (toddjordan) [inline bin/ember-fastboot]: k

**Comment by acorncom** [comment]
Nice work @Gaurav0 🎉

**Pull request merged**